juriscraper/opinions/united_states/state/coloctapp.py

```diff
@@ -35,14 +37,22 @@ def cleanup_content(content: str) -> str:
         :return: cleaned up html
         """
         tree = html.fromstring(content)
-        remove_xpaths = ["//style", "//img"]
-        for xpath in remove_xpaths:
-            if tree.xpath(xpath):
-                to_remove = tree.xpath(xpath)[0]
-                to_remove.getparent().remove(to_remove)
-
-        for tag in tree.xpath("//*[@class]"):
-            tag.attrib.pop("class")
+        remove_tags = ["//style", "//img"]
```

Comment:
Instead of doing this - perhaps we should modify 
from juriscraper.lib.html_utils import strip_bad_html_tags_insecure 

or some similar function to pass in extra tags we want to remove using Nh3 ?